helm/nessie/templates/_helpers.tpl

```diff
@@ -101,7 +101,7 @@ Apply Nessie Catalog (Iceberg REST) options.
 {{- $map := index . 1 -}}{{/* the destination map */}}
 {{- with $root -}}
 {{- $_ := set $map "nessie.catalog.default-warehouse" .defaultWarehouse -}}
-{{- $_ := set $map "nessie.catalog.object-stores.health-check.enabled" .objectStoresHealthCheckEnabled -}}
+{{- $__ := set $map "nessie.catalog.object-stores.health-check.enabled" .objectStoresHealthCheckEnabled -}}
```

Comment:
```suggestion
{{- $_ = set $map "nessie.catalog.object-stores.health-check.enabled" .objectStoresHealthCheckEnabled -}}
```

Reply:
IJ warns about a duplicate var here

Reply:
I don't have any warning. We are not declaring a new var (`:=`), we are reassigning `$_` (`=`).

Reply:
![image](https://github.com/projectnessie/nessie/assets/957468/d265efa1-b3ff-471a-8821-17878392a054)

Reply:
Please change `:=` to `=`